packages/config/config/devices/0x0460/qnsw-001P16.json

```
@@ -73,102 +99,220 @@
 		{
 			"#": "23",
 			"label": "O1 No/Nc",
-			"description": "0",
+			"description": "The set value determines the relay contact type for output O (O1). The relay contact type can be normally open (NO) or normally closed (NC).",
 			"valueSize": 1,
 			"minValue": 0,
 			"maxValue": 1,
 			"defaultValue": 0,
-			"unsigned": true
+			"unsigned": true,
+            "allowManualEntry": false,
+			"options": [
+				{
+					"label": "NO",
+					"value": 0
+				},
+				{
+					"label": "NC",
+					"value": 1
+				}
+			]
 		},
 		{
 			"#": "25",
 			"label": "Set Timer Units to Seconds Or Milliseconds For O1",
-			"description": "0",
+			"description": "Set the timer units to seconds or milliseconds. Choose if you want to set the timer in seconds or milliseconds in Parameters No. 19, 20.",
 			"valueSize": 1,
 			"minValue": 0,
 			"maxValue": 1,
 			"defaultValue": 0,
+			"unsigned": true,
+            "allowManualEntry": false,
+			"options": [
+				{
+					"label": "timer set in seconds",
+					"value": 0
+				},
+				{
+					"label": "timer set in milliseconds (scale 10ms)",
+					"value": 1
+				}
+			]
+		},
+        {
+			"#": "36",
+			"label": "O (O1) Power report on change - percentage",
+			"description": "This parameter determines the minimum change in consumed power that will result in sending a new report to the gateway.",
```

We usually call these kinds of parameters "Change Report Threshold". The unit makes it clear that this is a percentage.
```suggestion
			"label": "O1: Power Change Report Threshold",
```